### DISCUSSION TITLE
个人电脑配置

### DISCUSSION THREAD

**Discussion by carr123** [hpcaitech/ColossalAI#994] [Community | Q&A] [answered]
你好，听说 6GB 显存的RTX显卡就能训练15亿参数的模型哈。
我想配置台新台式机跑模型，8G显存左右。请问要跑15亿参数的模型，CPU, 内存等其他配置推荐多大好 ？
谢谢！

**Comment by binmakeswell** [comment, marked as answer]
建议32GB内存哈，加内存条成本相对低，在各方面体验会好很多。
